release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -3123,3 +3123,23 @@ link:https://access.redhat.com/solutions/6176142[{product-title} 4.6.36 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+=== RHBA-2021:2684 - {product-title} 4.6.39 bug fix
+
+Issued: 2021-07-21
+
+{product-title} release 4.6.39 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:2684[RHBA-2021:2684] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:2685[RHBA-2021:2685] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
```

Comment:
Suggest re-wording this a little for clarity; the infinitive form of "documenting" was a little tricky to my scanning 👀 :
Space limitations precluded complete documentation of the container images for this release. See the
link:https://access.redhat.com/solutions/6196661[{product-title} 4.6.39 container image list] for notes on the container images in this release.